components/function-controller/internal/controllers/serverless/runtime/runtime_test.go

```diff
@@ -59,21 +59,21 @@ func TestGetRuntimeConfig(t *testing.T) {
 				Runtime:                 serverlessv1alpha2.NodeJs16,
 				DependencyFile:          "package.json",
 				FunctionFile:            "handler.js",
-				DockerfileConfigMapName: "dockerfile-nodejs-16",
+				DockerfileConfigMapName: "dockerfile-nodejs16",
 				RuntimeEnvs: []corev1.EnvVar{
 					{Name: "FUNC_RUNTIME", Value: "nodejs16"}},
 			},
 		},
 		"default": {
-			name:    "nodejs14",
-			runtime: serverlessv1alpha2.NodeJs14,
+			name:    "nodejs12",
+			runtime: "default",
```

Comment:
The default shouldn't be a case anymore.

Reply:
Done after discussion with Filip